### PR TITLE
docs: add missing command and flags to CLI instructions

### PR DESCRIPTION
**Pull request by mdsolsawangwan**
I recently configured the `post-scan-action` plugin via `CLI` and ran into several issues:

- Was not able to create an `IAM` role with the instructed command. A `trust policy` is required so I've added this to step 4 in `CLI - Create an execution role for the Lambda function`.
- Was not able to subscribe to an `SNS` topic via `CLI`. I was able via the `AWS Console` only. It seems there's an additional command required, and so I've added it under step 3 in `CLI - Subscribe the Lambda to the SNS topic`

### CODE REVIEW

**Comment by mdsolsawangwan** [comment]
FYI: @trendmicro/cloud-one-file-storage-security

File: post-scan-actions/aws-python-promote-or-quarantine/README.md [hunk 1]
```diff
@@ -103,26 +103,46 @@ After a scan occurs, this example Lambda function places clean files in one buck
...
-    
+
    <details>
    <summary>Using the AWS CLI</summary>
 
-    1. Enter the following AWS CLI command to create the role:
+    1. Paste the [JSON code below](#JSON_trust-doc) into a file called `trust.json` (or another name).
+    2. Enter the following AWS CLI command to create the role:
 
         `aws iam create-role --role-name <YOUR_FSS_LAMBDA_ROLE>`
```
Comment (carpusherw) [inline post-scan-actions/aws-python-promote-or-quarantine/README.md:113]: Looks like you missed the step to use the `trust.json`. How about adding `--assume-role-policy-document` to `create-role` command?

**Review by carpusherw — CHANGES_REQUESTED** [review]

File: post-scan-actions/aws-python-promote-or-quarantine/README.md (hunk 1 above)
Comment (mdsolsawangwan) [reply post-scan-actions/aws-python-promote-or-quarantine/README.md]: @carpusherw ironic :) anyway good catch, I was working off a copy paste draft and forgot it but just pushed it. Thank you.

**Review by dougc-trend — DISMISSED** [review]
Looks good

**Review by carpusherw — COMMENTED** [review]
All good.
@mdsolsawangwan would you squash your commits to one, we want to maintain a clean history. Thanks!

**Comment by mdsolsawangwan** [comment]
@carpusherw done, thanks.

**Review by carpusherw — APPROVED** [review]
Merge Approval

**Pull request merged**